Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -213,7 +218,21 @@ CommandLineArgs::CommandLineArgs(const std::vector<std::wstring>& args)
             PrintUsage();
             return;
         }
-    }
+        else if ((_wcsicmp(args[i].c_str(), L"-ConcurrentLoad") == 0))
+        {
+            ToggleConcurrentLoad(true);
+        }
+        else if ((_wcsicmp(args[i].c_str(), L"-NumThreads") == 0))
+        {
+            unsigned num_threads = std::stoi(args[++i].c_str());
```

Comment:
If the user specifies -NumThreads without the required parameter, I think we would need to throw an error saying that the arguments are incorrect.

For example:
./winmlrunner -model ./squeezenet -ConcurrentLoad -NumThreads -CPU  should print an error indicating that there should be an integer argument next to NumThreads flag

Reply:
Agreed. We actually have this problem with other parameters too? I think it would be good for us to throw an exception in this function when such cases come up.

Reply:
Yes we have this problem with other parameters too. I agree we should throw an exception

Reply:
Yep yep! Agreed, we should throw exceptions for this function for other cases as well. Thanks Young!